config.default.yaml

```diff
@@ -340,18 +340,34 @@ costs:
 
 
 monte_carlo:
+  # Description: Specify Monte Carlo sampling options for uncertainty analysis.
+  # Define the option list for Monte Carlo sampling.
+  # Make sure add_to_snakefile is set to true to enable Monte-Carlo
   options:
-    add_to_snakefile: false
-    samples: 7    # number of optimizations
-    sampling_strategy: "chaospy"    # "pydoe2", "chaospy", "scipy", packages that are supported
-  pypsa_standard:
-      # User can add here flexibly more features for the Monte-Carlo sampling.
-      # Given as "key: value" format
-      # Key: add below the pypsa object for the monte_carlo sampling, "network" is only allowed for filtering!
-      # Value: currently supported format [l_bound, u_bound] or empty [], represent multiplication factors for the object
-    loads_t.p_set: [0.9, 1.1]
-    # generators_t.p_max_pu.loc[:, n.generators.carrier == "wind"]: [0.9, 1.1]
-    # generators_t.p_max_pu.loc[:, n.generators.carrier == "solar"]: [0.9, 1.1]
+    add_to_snakefile: false # When set to true, enables Monte Carlo sampling
+    samples: 9 # number of optimizations. Note that number of samples when using scipy has to be the square of a prime number
+    sampling_strategy: "chaospy"  # "pydoe2", "chaospy", "scipy", packages that are supported
+    seed: 42 # set seedling for reproducibilty
+
+  # Different distributions can be specified for various PyPSA network parameters.
+  # Supported distributions for uncertainties are uniform, normal, lognormal, triangle, beta and gamma.
+  # More info on the distributions are documented in the Chaospy reference guide...
+  # https://chaospy.readthedocs.io/en/master/reference/distribution/index.html
+  # ... users can add flexibly more features for the Monte-Carlo sampling
```

Comment:
it may be good to specify here how to clarify the parameters, in particular the following:
    {pypsa network object, e.g. "loads_t.p_set"}:
      type: {any distribution among: "uniform", "gaussian", ...}
      args: {arguments passed as a list depending on the distribution, see below supported ones}

Reply:
Thanks for the clarity @davide-f. I have included in the comments section more comments clarifying the parameters as suggested.

Feel free to review.